### DISCUSSION TITLE
ETIMEDOUT 20.221.80.166:443)

### DISCUSSION THREAD

**Discussion by Isaiasdevs** [community/community#70151] [Copilot Conversations]
### Select Topic Area

Bug

### Body

[ERROR] [ghostText] [2023-10-15T14:42:13.553Z] Error on ghost text request: (FetchError) The pending stream has been canceled (caused by: connect ETIMEDOUT 20.221.80.166:443)
 

what it is. 
two days with this problem

**Comment by Akash1134** [comment]
Hi @Isaiasdevs ,

These errors are usually caused by a corporate proxy setup that uses custom certificates to intercept and inspect secure connections. Can you please let us know whether you are accessing the Copilot behind the corporate firewall or proxy?

If yes, there are some possible ways to resolve certificate-related errors as documented over [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot#troubleshooting-certificate-related-errors).

One option that may work and I'd like you to try for testing is to add an [environment variable](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-network-settings-for-github-copilot#allowing-github-copilot-to-use-custom-certificates) `NODE_EXTRA_CA_CERTS` pointed to the exact path to a PEM file of the root CA certificate for your proxy's certificate. This isn't officially supported currently, but it will allow some setups to trust additional certificates.

Please review and let us know for any issues/queries.

Thanks!

**Reply by Isaiasdevs** [reply]
Thanks, I'm checking with my local provider. I already understand that there is a problem with them because on other networks the connection remains unstable. Very grateful for the contribution.

**Comment by Schranzfine** [comment]
Internal] Migrate FreewayToken FWT ERC20 (#26582)
* [Internal] Migrate FreewayToken FWT ERC20

Announcement: https://twitter.com/FreewayFi/status/1548050923397599234

Old: https://etherscan.io/token/0xf151980e7a781481709e8195744bf2399fb3cba4
https://etherscan.io/token/0x4A7397B0b86bB0f9482A3f4F16DE942f04E88702

New: https://etherscan.io/token/0x50d2CDd9E41D195a300569617cf9A662c63C37F3

* Auto fix all (sanity and consistency)

* Update info.json

* Fix coingecko
